appraisal.lic

```diff
@@ -96,11 +96,12 @@ class Appraisal
   end
 
   def appraise_item?(item)
-    @equipment_manager.get_item?(item)
-    bput("appraise #{item.short_name} quick", 'Roundtime')
-    waitrt?
-    pause 1
-    @equipment_manager.empty_hands
+    if @equipment_manager.get_item?(item)
+      bput("appraise #{item.short_name} quick", 'Roundtime')
+      waitrt?
+      pause 1
+      @equipment_manager.empty_hands
```

Comment:
These 4 lines are now indented within the IF statement, no other change to the logic

Reply:
Thank you for pointing that out